eclass/rpm-extended.eclass

```diff
@@ -0,0 +1,76 @@
+# Copyright 1999-2020 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
```

Comment:
add `@ECLASS`, `@MAINTAINER` and basically all the things from the header from the `python-single-r1` eclass.